cmd/state-installer/internal/installer/installation_darwin.go

```diff
@@ -9,42 +9,27 @@ import (
 	"github.com/ActiveState/cli/internal/appinfo"
 	"github.com/ActiveState/cli/internal/errs"
 	"github.com/ActiveState/cli/internal/fileutils"
-	"github.com/gobuffalo/packr"
 )
 
 const (
-	appDir  = "/Applications"
 	appName = "ActiveState Desktop.app"
 )
 
 // InstallSystemFiles installs files in the /Application directory
-func InstallSystemFiles(installDir string) error {
-	err := os.RemoveAll(filepath.Join(appDir, appName))
+func InstallSystemFiles(fromDir, binaryDir, systemInstallPath string) error {
+	err := os.RemoveAll(filepath.Join(systemInstallPath, appName))
```

Comment:
This would produce an error if doesn't exist, I think?

Reply:
No, `os.RemoveAll` will return nil if the path doesn't exist.